modules/plot/main/coffee/Axis.coffee

```
@@ -1,12 +1,22 @@
 
 # by far the most complicated component of the graphing api. The auto axis scaling makes things really complicated here...
+doCollisionDetection = (nodes) ->
+  nodes.forEach (node, i) ->
+    previousNodes = nodes.slice 0, i
+    doesOverlap = hx.find previousNodes, (previousNode) ->
```

hx.find should fail early